.ci_support/migrations/cuda120.yaml

```
@@ -66,20 +62,20 @@ __migrator:
     [see this issue]( https://github.com/conda-forge/conda-forge.github.io/issues/1963 ).
     Please feel free to raise any issues encountered there. Thank you! :pray:
 
-cuda_compiler:                 # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - cuda-nvcc                  # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+cuda_compiler:                 # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - cuda-nvcc                  # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 
-cuda_compiler_version:         # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12.0                       # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+cuda_compiler_version:         # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 12.0                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 
 c_compiler_version:            # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 11                         # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 12                         # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Think we still need to use GCC 11 here

```suggestion
  - 11                         # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```